info/v1/container.go

```diff
@@ -296,6 +296,14 @@ type CpuUsage struct {
 	// Time spent in kernel space.
 	// Unit: nanoseconds.
 	System uint64 `json:"system"`
+
+	// CPU time consumed per core in kernel mode
+	// Units: nanoseconds.
+	PercpuUsageInKernelmode []uint64 `json:"percpu_usage_in_kernelmode"`
```

Comment:
Consider shorter name of variable, e.g. `PerCpuKernel` and short json key, e.g: "json:percpu_usage_kernel" .
I think that it fits better to the rest of fields in CpuUsage struct, see: [PerCpu](https://github.com/wacuuu/cadvisor/pull/3/files#diff-4e7bb0420ef22ca45ccd056f4687f8feR290)

Reply:
I still think that it is better to have shorter names and have the same names in `info/v2/container.go`.
Why have you decided to change it?